lib/js/components/Drawer/DrawerHeader/DrawerHeader.vue

```diff
@@ -129,6 +138,11 @@ $minimal-drawer-header-height: 82px;
 		padding: $space-m $space-xs $space-m $space-s;
 	}
 
+	&__actions {
+		align-self: flex-start;
```

Comment:
Is misaligned with close icon

Reply:
I think we can remove align-self here and remove __closeButton class, and it will be as in Figma